include/MixerChannelView.h

```diff
@@ -100,6 +105,8 @@ namespace lmms::gui
 
     private:
         QString elideName(const QString& name);
+        MixerChannel* mixerChannel() const;
+        inline auto isMasterChannel() const -> bool { return m_channelIndex == 0; }
```

Comment:
When my suggestion put the definitions into the header, that was my way of saying to inline the functions, because for functions defined inside class declarations, they are inlined by default.

Reply:
So shall I remove the explicit `inline` statement from `isMasterChannel`?

Reply:
Yeah, its redundant.

Reply:
Done with commit 611df8e3d04.